### PR TITLE
Updated default SnapRAID version to 8.1

### PR DESCRIPTION
**Pull request by Sam-Martin**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@
 license          'Apache v2.0'
 description      'Installs/Configures snapraid'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.3.0'
+version          '0.3.1'
+supports		'ubuntu'
```
Comment (X0nic) [inline metadata.rb]: Some strange formatting here

**Comment by X0nic** [comment]
Just one nit. A little bit of strange formatting, but looks good other than that.

**Comment by Sam-Martin** [comment]
Whoops, tabs vs spaces, need to fix my sublime config!

**Comment by X0nic** [comment]
Sorry about that! I forgot to come back, and merge it.

**Comment by Sam-Martin** [comment]
No worries!
Do you fancy pushing it up to the Supermarket? (I've already updated it to
0.3.1 from 0.3.0)

Thanks,

Sam

**Comment by X0nic** [comment]
Sorry @Sam-Martin I missed this. I will push up a new version shortly

**Comment by Sam-Martin** [comment]
Awesome! :+1:

**Pull request merged**